### DISCUSSION TITLE
Remove dependency on React

### DISCUSSION THREAD

**Discussion by davetapley** [ConrabOpto/mst-query#34] [Ideas]
Great idea for a library @k-ode, but are you open to removing dependency on React?

See also: https://github.com/ryansolid/mobx-jsx

**Comment by k-ode** [comment]
@davetapley Yes, this is a long term goal of this library. Do you want to use it with mobx-jsx? Would be great to have concrete use case to test against.

**Reply by davetapley** [reply]
We have a moderately large React codebase which I'm slowly replacing parts of with [MST. ](https://mobx-state-tree.js.org/)
Probably a few years ago, and just scoping out viable replacements.
